src/legacy/core_plugins/kibana/index.js

```diff
@@ -85,7 +85,7 @@ export default function (kibana) {
           }),
           order: -1003,
           url: `${kbnBaseUrl}#/discover`,
-          icon: 'plugins/kibana/assets/discover.svg',
+          icon: 'plugins/kibana/discover/assets/discover.svg',
```

Comment:
@flash1293 @timroes I've migrated/moved the assets, but I think we can remove them, it looks like `.icon` was the old way our sidemenu icons where rendered. From my point of view they are still used as a fallback, when no EUI icon was present. However, all icons are euficated now.

https://github.com/elastic/kibana/blob/41aea39cc06b391eb142a937e350337799c528ee/src/core/public/chrome/ui/header/header.tsx#L341

Reply:
Agreed - it also looks like the icons are not up to date anymore. The fallback logic should stay in `header.tsx` though to support 3rd party plugins like this.

Reply:
Ok thanks, so I will clean this up!